src/Reflex/Dom/Builder/Static.hs

```diff
@@ -176,6 +177,7 @@ instance SupportsStaticDomBuilder t m => DomBuilder t (StaticDomBuilderT t m) wh
       , _inputElement_hasFocus = hasFocus
       , _inputElement_element = e
       , _inputElement_raw = ()
+      , _inputElement_files = constDyn mempty
```

Comment:
For future reference, constDyn mempty == mempty; however, this is also fine, so no need to change it here.